processor/coralogixprocessor/metadata.yaml

```
@@ -0,0 +1,10 @@
+type: coralogix
+
+status:
+  class: processor
+  stability:
+    development: [traces]
+  distributions: [contrib]
```

```suggestion
  distributions: []
```
We shouldn't mark it as being in `contrib` until it's included in `contrib` releases. Otherwise, it's easy for users to get confused and wonder why they can't use it with released `contrib` images.